doc/reference.md

```diff
@@ -655,13 +717,17 @@ Methods used to provide payment during a successful checkout flow:
 
 ## Errors
 
-Error                                                | Cause                                                               | Returned by
----------------------------------------------------- | ------------------------------------------------------------------- | ---
-<a id="e1">`usageError`</a>                          | Reader SDK was used in an unexpected or unsupported way.            | all methods
-<a id="e2">`authorizeErrorNoNetwork`</a>             | Reader SDK could not connect to the network.                        | [authorize](#authorize)
-<a id="e3">`checkoutErrorCanceled`</a>               | The user canceled the checkout flow.                                | [startCheckout](#startcheckout)
-<a id="e4">`checkoutErrorSdkNotAuthorized`</a>       | The checkout flow started but Reader SDK was not authorized.        | [startCheckout](#startcheckout)
-<a id="e5">`readerSettingsErrorSdkNotAuthorized`</a> | The Reader settings flow started but Reader SDK was not authorized. | [startReaderSettings](#startreadersettings)
+Error                                                    | Cause                                                               | Returned by
+-------------------------------------------------------- | ------------------------------------------------------------------- | ---
+<a id="e1">`usageError`</a>                              | Reader SDK was used in an unexpected or unsupported way.            | all methods
+<a id="e2">`authorizeErrorNoNetwork`</a>                 | Reader SDK could not connect to the network.                        | [authorize](#authorize)
+<a id="e3">`checkoutErrorCanceled`</a>                   | The user canceled the checkout flow.                                | [startCheckout](#startcheckout)
+<a id="e4">`checkoutErrorSdkNotAuthorized`</a>           | The checkout flow started but Reader SDK was not authorized.        | [startCheckout](#startcheckout)
+<a id="e5">`readerSettingsErrorSdkNotAuthorized`</a>     | The Reader settings flow started but Reader SDK was not authorized. | [startReaderSettings](#startreadersettings)
+<a id="e6">`storeCustomerErrorCanceled`</a>              | The user canceled the store card flow.                              | [startStoreCard](#startstorecard)
+<a id="e7">`storeCustomerErrorInvalidCustomerId`</a>     | The customer ID passed into the controller was invalid.             | [startStoreCard](#startstorecard)
+<a id="e8">`storeCustomerErrorSdkNotAuthorized`</a>      | Reader SDK was not authorized.                                      | [startStoreCard](#startstorecard)
```

Comment:
Per comments on the other PR: we should differentiate the various errors around Reader SDK not being authorized based on the situation in which the error is raised. Suggest:

The flow to save a customer card started but Reader SDK was not authorized.